desktop/ui/src/main/java/org/datacleaner/connection/DatastoreDescriptorDesktopBindings.java

```diff
@@ -0,0 +1,186 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.connection;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.datacleaner.database.DatabaseDriverCatalog;
+import org.datacleaner.util.IconUtils;
+import org.datacleaner.windows.AbstractDatastoreDialog;
+import org.datacleaner.windows.AccessDatastoreDialog;
+import org.datacleaner.windows.CassandraDatastoreDialog;
+import org.datacleaner.windows.CompositeDatastoreDialog;
+import org.datacleaner.windows.CouchDbDatastoreDialog;
+import org.datacleaner.windows.CsvDatastoreDialog;
+import org.datacleaner.windows.DbaseDatastoreDialog;
+import org.datacleaner.windows.ElasticSearchDatastoreDialog;
+import org.datacleaner.windows.ExcelDatastoreDialog;
+import org.datacleaner.windows.FixedWidthDatastoreDialog;
+import org.datacleaner.windows.HBaseDatastoreDialog;
+import org.datacleaner.windows.JdbcDatastoreDialog;
+import org.datacleaner.windows.JsonDatastoreDialog;
+import org.datacleaner.windows.MongoDbDatastoreDialog;
+import org.datacleaner.windows.SalesforceDatastoreDialog;
+import org.datacleaner.windows.SasDatastoreDialog;
+import org.datacleaner.windows.SugarCrmDatastoreDialog;
+import org.datacleaner.windows.XmlDatastoreDialog;
+
+public class DatastoreDescriptorDesktopBindings {
```

Comment:
I like this class :-) And I think the DatastoreDescriptor should be closer tied to it - it's a (desktop) UI thing only right now, so I would argue that they should both be in the desktop module. Furthermore I would find it pretty natural that this class had a couple of methods to help layout like "getDatabaseDatastoreDescriptors", "getCloudServiceDatastoreDescriptors" and "getFileDatastoreDescriptor". That would eliminate the need for the tags system.